spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
+Dir[ManageIQ::API::Common::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
```

Comment:
You don't need this line. `RequestSpecHelper` does the same thing as `UserHeaderSpecHelper` now.

Reply:
Removed it for now, until we can debug why the common gem `spec/support` isn't loading properly.